packages/builder/src/contracts/deployments.ts

```diff
@@ -1,20 +1,25 @@
-export const chains: { [key: number]: string } = {
-  31337: "localhost",
+import { ChainId } from "common";
+
+export const chains = {
+  1: "mainnet",
   5: "goerli",
   10: "optimism",
-  69: "optimisticKovan", // todo: update to 420: "optimisticGoerli"
```

Comment:
why are we removing optimism testnets?

Reply:
Kovan is deprecated

Reply:
We can add support for optimistic goerli, but I feel like we're bloating the scope of this PR a bit already, so would leave that for another one.

Reply:
ok, but just for the next time, if it's not related to the PR, don't even remove the deprecated one. if you remove it you need to add the new one